Error: Cannot find module with sequelize in 1.1.1

When running `./node_modules/.bin/sequelize db:migrate` I get `Error: Cannot find module 'terraformer-wkt-parser'`

If I force terraformer-wkt-parser to version `1.1.0` there is no error.

--- a/.gitignore
+++ b/.gitignore
@@ -2,9 +2,13 @@ node_modules
 coverage
 SpecRunner.html
 .grunt
-dist
+
 npm-debug.log
 
 # TypeScript
 typings/
 test.js
+
+# built files
+terraformer-wkt-parser.js
+terraformer-wkt-parser.min.js

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -18,8 +18,8 @@ module.exports = function (grunt) {
         banner: '<%= meta.banner %>'
       },
       wkt: {
-        src: ["dist/terraformer-wkt-parser.js"],
-        dest: 'dist/terraformer-wkt-parser.min.js'
+        src: ["terraformer-wkt-parser.js"],
+        dest: 'terraformer-wkt-parser.min.js'
       }
     },
 
@@ -38,7 +38,7 @@ module.exports = function (grunt) {
     jasmine: {
       coverage: {
         src: [
-          "dist/terraformer-wkt-parser.js"
+          "terraformer-wkt-parser.js"
         ],
         options: {
           specs: 'spec/*Spec.js',
@@ -91,7 +91,7 @@ module.exports = function (grunt) {
       dev: {
         upload: [
           {
-            src: 'dist/terraformer-wkt-parser.min.js',
+            src: 'terraformer-wkt-parser.min.js',
             dest: 'terraformer-wkt-parser/<%= pkg.version %>/terraformer-wkt-parser.min.js'
           }
         ]
@@ -118,7 +118,7 @@ module.exports = function (grunt) {
 
     wrapper = wrapper.replace('"SOURCE";', parserSource);
 
-    fs.writeFileSync("./dist/terraformer-wkt-parser.js", wrapper, "utf8");
+    fs.writeFileSync("./terraformer-wkt-parser.js", wrapper, "utf8");
 
     grunt.log.write('Files created.\n');
   });

--- a/test/convert-test.js
+++ b/test/convert-test.js
@@ -1,6 +1,6 @@
 var vows    = require('vows'),
     assert  = require('assert'),
-    wkt     = require('../dist/terraformer-wkt-parser');
+    wkt     = require('../terraformer-wkt-parser');
 
 
 var point      = require('../examples/point.json');

--- a/test/terraformer-parse-test.js
+++ b/test/terraformer-parse-test.js
@@ -1,6 +1,6 @@
 var vows    = require('vows'),
     assert  = require('assert'),
-    wkt     = require('../dist/terraformer-wkt-parser');
+    wkt     = require('../terraformer-wkt-parser');
 
 var fs = require('fs');
 

--- a/test/wkt-parse-test.js
+++ b/test/wkt-parse-test.js
@@ -1,6 +1,6 @@
 var vows    = require('vows'),
     assert  = require('assert'),
-    wkt     = require('../dist/terraformer-wkt-parser');
+    wkt     = require('../terraformer-wkt-parser');
 
 var fs = require('fs');
 

